common/resources.yaml

```diff
@@ -0,0 +1,49 @@
+datasets:
+ - data_type_def: ResourceReferences
+   doc: A table for identifying which objects in a file contain
+        values that correspond to external resource references. An
+        external resource reference takes the form of an *id*, which
+        is resolvable within a reference. Some of examples are
+        GO:0005524, an entry in the Genome Ontology, ABE-0008876, an entry
+        in the ASAP microbial genome database, or MGI:88294, an entry in the
+        Mouse Genome Informatics database.
+   dtype:
+    - name: id
+      dtype: uint
+      doc: the unique identifier in this table
+    - name: object_id
+      dtype: text
+      doc: the UUID for the object that references the resourece
+    - name: field
+      dtype: text
+      doc: the field from the object (specified by object_id) that contains the reference
+    - name: item
+      dtype: uint
+      doc: an index into the ResourceIdentiferMap that contains the term
+ - data_type_def: ResourceIdentiferMap
```

Comment:
@ajtritt do you think it would be useful to set ``datatype_inc: VectorData`` for the ``ResourceIdentifierMap`` and ``ResourceReferences``. In this way they one could more easily reuse these tables and embed them in a DynamicTable if necessary in the future (e.g., as part of an extension).